docs/guides/ipsec-guide.md

```diff
@@ -0,0 +1,129 @@
+# IPsec Offload Guide
```

Comment:
This is no longer a standalone Guide, it's a chapter in the Features section of the user guide. Its title should therefore be the name of the feature. In this case, I assume it's _IPsec Offload_.